### PR TITLE
Angular 17 Course added and material added

### PR DESCRIPTION
**Pull request by Vinoth2626**

### CODE REVIEW

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -17,6 +18,7 @@ export const routes: Routes = [
       { path: 'dashboard', component: DashboardComponent },
       { path: 'program', component: ProgramComponent },
       { path: 'header', component: HeaderComponent },
+      {path:'fooder',component:FooderComponent}
```
Comment (thangaraj4747) [inline src/app/app.routes.ts]: use prettier here

File: src/app/courses/course-card/course-card.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <div class="course-title">
+    {{ course.description }}
+  </div>
+  <br />
+  <!--   
+    <ng-content select="course-image"
+                *ngIf="course.iconUrl"></ng-content> -->
+  @if(course.iconUrl){
+  <img width="250" alt="algular-Logo" [src]="course.iconUrl" />
+  } @else { No Image available! }
```
Comment (thangaraj4747) [inline src/app/courses/course-card/course-card.component.html]: i guess need to add some tag here 'No Image available!'

Comment (Vinoth2626) [reply src/app/courses/course-card/course-card.component.html]: Issue resolved

File: src/app/app.routes.ts (hunk 1 above)
Comment (Vinoth2626) [reply src/app/app.routes.ts]: issue resoved

**Review by thangaraj4747 — APPROVED** [review]

**Pull request merged**